### DISCUSSION TITLE
Multiple Windows in ScollArea

### DISCUSSION THREAD

**Discussion by vbeffa** [emilk/egui#1034] [Q&A]
I have multiple tables, each of which is laid out in a Window, and I need to display all of them always expanded (not collapsible). They take up more space than is available vertically on the screen, so I need to add a Scrollarea at the top level (like, inside a CentralPanel), and then add Windows inside there.

Is it possible to do this?

**Comment by emilk** [comment]
Windows cannot be put inside a `Ui`. That's why they take a `CtxRef` when you show them, not a `Ui`: https://docs.rs/egui/latest/egui/containers/struct.Window.html#method.show
